fab

```diff
@@ -339,11 +363,10 @@ Configuration environment variables:
         "cpp",
         description="Preprocess a plan",
         formatter_class=RawDescriptionHelpFormatter,
-        epilog="""note:
-    cpp options are not taken exactly as `cpp` would. For `-I<path>` is invalid,
-    you must do `-I=<path>` or `-I <path>`
-        
-See cpp(1) man page for further details""",
+        epilog=("Note: cpp options are not taken exactly as `cpp` would.\n"
+                "      For example: `-I<path>` is invalid, youu must instead"
```

Comment:
Typo:

`you` got turned to `youu`

Reply:
Good find! I searched and searched and couldn't see it until I read your comment... Good find.